tests/positive/VP/PolySimpleFungibleToken.mjuvix

```diff
@@ -0,0 +1,309 @@
+module PolySimpleFungibleToken;
+
+foreign ghc {
+  import Anoma
+};
+
+--------------------------------------------------------------------------------
+-- Booleans
+--------------------------------------------------------------------------------
+
+inductive Bool {
+  true : Bool;
+  false : Bool;
+};
+
+infixr 2 ||;
+|| : Bool → Bool → Bool;
+|| false a ≔ a;
+|| true _ ≔ true;
+
+infixr 3 &&;
+&& : Bool → Bool → Bool;
+&& false _ ≔ false;
+&& true a ≔ a;
+
+if : (A : Type) → Bool → A → A → A;
+if _ true a _ ≔ a;
+if _ false _ b ≔ b;
+
+--------------------------------------------------------------------------------
+-- Backend Booleans
+--------------------------------------------------------------------------------
+
+axiom BackendBool : Type {
+  ghc ↦ "Bool";
+};
+
+axiom backend-true : BackendBool {
+  ghc ↦ "True";
+};
+
+axiom backend-false : BackendBool {
+  ghc ↦ "False";
+};
+
+--------------------------------------------------------------------------------
+-- Backend Bridge
+--------------------------------------------------------------------------------
+
+foreign ghc {
+  bool :: Bool -> a -> a -> a
+  bool True x _ = x
+  bool False _ y = y
+};
+
+axiom bool : BackendBool → Bool → Bool → Bool {
+  ghc ↦ "bool";
+};
+
+from-backend-bool : BackendBool → Bool;
+from-backend-bool bb ≔ bool bb true false;
+
+--------------------------------------------------------------------------------
+-- Functions
+--------------------------------------------------------------------------------
+
+const : (A : Type) → (B : Type) → A → B → A;
+const _ _ a _ ≔ a;
+
+id : (A : Type) → A → A;
+id _ a ≔ a;
+
+--------------------------------------------------------------------------------
+-- Integers
+--------------------------------------------------------------------------------
+
+axiom Int : Type {
+  ghc ↦ "Int";
+};
+
+axiom lt : Int → Int → BackendBool {
```

Comment:
Use the new syntax for compile blocks.

Reply:
fixed in 9441795e865853020c749b0cc10e0f0f3cfda7bb.
I have also added [PolySimpleFungibleToken.mjuvix](https://github.com/heliaxdev/minijuvix/pull/70/files/84a4850d02358de80be2e8ecb9357e7c42390397#diff-ea297effdeac2854253d51dcec17553769813c11ab4e3df342d2c6771a08b5c2) to the test suite